launcher

```diff
@@ -259,6 +260,15 @@ check_prereqs() {
     echo "WARNING: Git version ${test} deprecated, recommend upgrade to ${git_rec_version} or newer."
   fi
 
+  # Check minimum kernel version due to https://bugs.ruby-lang.org/issues/13885
+  test=($(uname -r))
```

Comment:
Can launcher be… launched on non-linux platforms? Should it limit that check to linux only?

Reply:
Considering launcher is used to run production Discourse installs, and we only support production in Linux I don't see why we should support running it on non-Linux OSs.